src/Nethermind/Nethermind.Serialization.Rlp/RlpBehaviors.cs

```diff
@@ -28,7 +28,10 @@ public enum RlpBehaviors
         Storage = 4,
         Eip658Receipts = 8,
         AllowUnsigned = 16,
-        ForTreeRoot = 32,
-        All = AllowExtraData | ForSealing | Storage | Eip658Receipts | AllowUnsigned | ForTreeRoot
+        SkipTypedWrapping = 32, // introduced after typed transactions. In the network (devp2p) transaction has additional wrapping
```

Comment:
Can we have 2 flags there with same value?
SkipTypedWrapping = 32
and 
ForTreeRoot = 32

yes they are the same, but intention differs...

Not sure if good idea.

Reply:
Hmmmm. IMO the intention is the same. We're skipping byte array rlp for typed objects.

Reply:
Maybe name it:
ForTreeRoot 
ForRpc ?